test/test_helper.rb

```diff
@@ -283,6 +283,17 @@ def default_fixture_filename
         "#{fixture_prefix}_romsey"
       end
     end
+
```

Comment:
@mdebo The tests have failed due to the merge of #970.
To fix them, add `require 'geocoder/lookups/geoportail_lu'` here.